src/main/java/io/github/syst3ms/skriptparser/expressions/CondExprIsEmpty.java

```diff
@@ -1,56 +1,54 @@
 package io.github.syst3ms.skriptparser.expressions;
 
-import java.util.Arrays;
-
-import org.jetbrains.annotations.Nullable;
-
 import io.github.syst3ms.skriptparser.Parser;
 import io.github.syst3ms.skriptparser.lang.Expression;
 import io.github.syst3ms.skriptparser.lang.TriggerContext;
 import io.github.syst3ms.skriptparser.lang.base.ConditionalExpression;
 import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Arrays;
 
 /**
  * Check if a given string or list is empty.
+ * Note that multiple strings will all be checked for blankness.
+ * If you want a string list to be checked as a list instead of all separate strings,
+ * you should use the 'list' modifier at the end.
  *
- * @name Empty
- * @pattern %strings/objects% (is|are)[1:( not|n't)] empty
+ * @name Is Empty
+ * @pattern %strings% (is)[1:( not|n't)] empty
+ * @pattern %objects% (is|are)[1:( not|n't)] empty
  * @since ALPHA
  * @author Olyno
  */
-public class CondExprEmpty extends ConditionalExpression {
-
+public class CondExprIsEmpty extends ConditionalExpression {
     static {
-        Parser.getMainRegistration().addExpression(CondExprEmpty.class,
-            Boolean.class,
-            true,
-            2,
-            "%strings/objects% (is|are)[1:( not|n't)] empty"
+        Parser.getMainRegistration().addExpression(CondExprIsEmpty.class,
+                Boolean.class,
+                true,
+                5,
+                "%strings% (is|are)[1:( not|n't)] empty",
+                "%objects% is[1:( not|n't)] [an] empty list"
```

Comment:
I believe the two syntaxes should instead be:
```autohotkey
%objects% (is|are)[1:( not|n't)] empty
%strings% (is|are)[1:( not|n't)] [an] empty string[s]
```
for two reasons: 

1. Checking if a list is empty is too important to have boilerplate associated with it
2. If the user submits a list of strings to the first syntax, it makes more sense IMO to check if the _list_ isn't empty rather than the strings. List of strings are a common thing, much more common than checking if multiple strings are all empty.

In case it wasn't clear, the first syntax should check for list emptiness if there are multiple elements (even if they are all strings), and string emptiness if there is only one element. However, the second syntax should always check for string emptiness.